general/bundler_imports.md

```diff
@@ -0,0 +1,160 @@
+# Bundler Imports
+
+This document is aimed at explaining how npm bundler imports work.
+
+## Why do we need npm bundler imports
```

Comment:
If it is a question, end with "?"

Otherwise, rephrase to not be a question: eg. "Why we need npm bundler imports".